huridocs/uwazi
# #992 - Fix all Settings buttons on footer

Fixes issue #992

PR check list:


QA check list:

- [ ] Smoke test the functionality described in the issue

NOTE: Make sure the build passes.
